dust_extinction/averages.py

```diff
@@ -20,6 +20,7 @@
     "CT06_MWLoc",
     "GCC09_MWAvg",
     "F11_MWGC",
+    "B1992"
```

Comment:
The convention I've been using would make the name for this model `B92_MWAvg`.  Check out https://dust-extinction.readthedocs.io/en/latest/dust_extinction/references.html#naming-convention.  Please also add the appropriate reference to that documentation page.

Reply:
Also move the name to right the 2nd in the list.  See the comment below.